src/expense/expenses.controller.js

```diff
@@ -0,0 +1,106 @@
+'use strict';
+
+const {
+  getAllExpenses,
+  getOneExpense,
+  checkAtleastOneUser,
+  createExpense,
+  findIndexOneExpense,
+  deleteExpense,
+  updateExpense,
+} = require('../services/expenses.service');
+
+const getAll = (req, res) => {
+  const { userId, categories, from, to } = req.query;
+
+  const expensesQuery = getAllExpenses(userId, categories, from, to);
+
+  res.send(expensesQuery);
+};
+
+const getOneById = (req, res) => {
+  const { id } = req.params;
+
+  const expenseIdFound = getOneExpense(id);
+
+  if (!expenseIdFound) {
+    res.status(404).send('Expense not found');
```

Comment:
It might be better to add all status codes and messages to constants.

Reply:
Agreed. Create constants for all status codes.